.style-dictionary/filesGenerators.ts

```diff
@@ -0,0 +1,69 @@
+import StyleDictionary from 'style-dictionary'
+import { breakpointsEnum } from '../src/tokens/breakpoint'
+const { fileHeader } = StyleDictionary.formatHelpers;
+
+export function generateTokensFile() {
+  StyleDictionary
+  .registerTransform({
+    name: 'name/dashed',
+    type: 'name',
+    transformer: (prop) => prop.path.join('-'),
+  })
+  .extend({
+    source: ['.style-dictionary/tokens.ts'],
+    platforms: {
+      css: {
+        transforms: [
+          'attribute/cti',
+          'name/dashed',
+          'time/seconds',
+          'content/icon',
+          'size/rem',
+          'color/css',
+        ],
```

Comment:
```suggestion
      transformGroup: 'css',
```

https://amzn.github.io/style-dictionary/#/transform_groups?id=css